### PR TITLE
191 open file location bug

### LINKED ISSUE #191: Bug: "Open File Location" does nothing
At the end of processing an image, a modal pops up with a button to "Open File Location". Currently the button does nothing.

AC:
- When user clicks the button, open the file explorer(?) to the location of the output folder that contains the color corrected image .

Code Location:
Image

### PR DESCRIPTION
**Pull request by rfl8484**
Resolves #191 
Fixed the "Open File Location" button at the end of Processing.

Will open the file manager using the electron shell module. Added in a new ipc handler, ipc-openFileExplorer. After the image is finished processing, the backend returns the filepath of the .btrgb file. I used regex to take off the file portion of the filepath and end up with just the directory it is located in.

First iteration:
Used the ipc-Dialog handler function to open the file explorer, but the file explorer is a child process of BeyondRGB, not its own process, and is intended to enable the user to select files/directories (like in the processing steps). I added in the optional 'defaultPath' parameter to open it in a specified directory. Also added this attribute to FileSelector.svelte.

Second Iteration:
Using the electron shell to start a new file explorer process allows the file explorer to persist even after BeyondRGB has been closed out. Decided to leave in modifications to FileSelector.svelte in case they are needed in future and they don't break anything (lmk if I should remove them).

### CODE REVIEW

File: frontend/src/renderer/components/Process/Tabs/Processing.svelte [hunk 1]
```diff
@@ -52,22 +52,35 @@
...
+  }
 </script>
 
 <main>
   {#if $processState.pipelineComplete}
     <div class="completedBox">
       <div class="completedOptions">
         <button on:click={() => handleComplete(0)}>View Image</button>
-        <button disabled on:click={() => handleComplete(1)}
+        <button on:click={async () => {await openFileExplorer();}}
```
Comment (keenanm500) [inline frontend/src/renderer/components/Process/Tabs/Processing.svelte:80]: I think the onClick could just be this: `on:click={openFileExplorer}`

File: frontend/src/renderer/components/Process/Tabs/Processing.svelte [hunk 2]
```diff
@@ -52,22 +52,35 @@
...
           >Open File Location</button
         >
-        <button on:click={() => handleComplete(2)}>Process Another Image</button
+        <button on:click={() => handleComplete(1)}>Process Another Image</button
```
Comment (keenanm500) [inline frontend/src/renderer/components/Process/Tabs/Processing.svelte:83]: No need to change here, but these "handleComplete" numbers should probably be enums or something

**Review by keenanm500 — APPROVED** [review]
Glad someone got around to this

**Review by jep7453 — APPROVED** [review]
Tested on my machine, feature works. Other comments have some possible changes, but they don't seem fully necessary. Looks good to me.

**Pull request merged**